packages/core/src/lib/utils.ts

```
@@ -1,12 +1,13 @@
 import { BitcoinNetwork, IBaseWallet } from '@catalogfi/wallets';
-import { Environment, with0x } from '@gardenfi/utils';
+import { AsyncResult, Environment, Err, Ok, with0x } from '@gardenfi/utils';
```

ESLint

[error] 'AsyncResult' is defined but never used.

(@typescript-eslint/no-unused-vars)